src/tools/run-make-support/src/symbols.rs

```diff
@@ -2,18 +2,32 @@ use std::path::Path;
 
 use object::{self, Object, ObjectSymbol, SymbolIterator};
 
-/// Iterate through the symbols in an object file.
-///
-/// Uses a callback because `SymbolIterator` does not own its data.
+/// Given an [`object::File`], find the exported dynamic symbol names via
+/// [`object::Object::exports`]. This does **not** impose any filters on the specific dynamic
+/// symbols, e.g. if they are global or local, if they are defined or not, and in which section the
```

Comment:
I don't know about other formats but at least on ELF `exports()` appears to only return defined symbols

Reply:
Fixed comment